src/cluster-configuration/deploy/gpu-configuration/gpu-configuration.json.template

```diff
@@ -1,9 +1,9 @@
 {
   "nodes" : {
-{% for host in cluster_cfg['layout']['machine-list'] if 'gpu' in cluster_cfg['layout']['machine-sku'][cluster_cfg['layout']['machine-list'][host]['machine-type']] %}
```

Comment:
What if an old cluster is updated to this version and don't have `computing-device` in `layout.yaml`?

I think we may add some defaulting codes in `paictl` to handle legacy issues.

Reply:
> What if an old cluster is updated to this version and don't have `computing-device` in `layout.yaml`?
> 
> I think we may add some defaulting codes in `paictl` to handle legacy issues.

As we are keep involving breaking changes, we will handle this issue before code-freeze.  This issue will be tracked in #5100 .